client/src/main/java/org/spine3/change/MessageMismatch.java

```diff
@@ -70,6 +70,22 @@ public static ValueMismatch expectedNotDefault(Message expected, int version) {
     }
 
     /**
+     * Creates a {@code ValueMismatch} for a command that wanted to <em>change</em> a field value,
+     * but discovered that the field has the default value.
+     *
+     * @param expected the value expected by the command
+     * @param newValue the value the command wanted to set
+     * @param version  the version of the entity in which the mismatch is discovered
+     * @return new {@code ValueMismatch} instance
+     */
+    public static ValueMismatch expectedNotDefault(Message expected, Message newValue, int version) {
+        checkNotNull(expected);
```

Comment:
I thought we treat the method parameters as non-nullable due to the annotation in `package-info.java`. If that's still true, I'd think that `checkNotNull` statements should be removed.

If someone passes `null` values to this method, he will get an NPE (or maybe something even worse) for violating an API contract.